roles/openshift_default_storage_class/defaults/main.yml

```diff
@@ -13,6 +13,12 @@ openshift_storageclass_defaults:
     parameters:
       type: pd-standard
 
+  openstack:
+    name: sc-cinder
```

Comment:
actually I think we should call it 'standard' to be consistent with upstream kube. I think that's why the aws storageclass is called 'gp2' and the gce one 'standard', because we are copying upstream. https://github.com/kubernetes/kubernetes/blob/3e694c38e0b423b1c79a0d7c69f74137a357653f/cluster/addons/storage-class/openstack/default.yaml

Reply:
Thanks @wongma7, submit a new commit to change StorageClass name to standard.